layers/+tools/terraform/packages.el

```diff
@@ -12,6 +12,8 @@
 (setq terraform-packages
       '(
         (company-terraform :requires company)
+        (lsp-terraform :requires lsp-mode
```

Comment:
This package cannot be installed, instead it is part of `lsp-mode`.
Please remove it from here and instead modify `terraform/init-terraform-mode` to setup the mode hook to startup lsp.

See the following excerpt from the python layer:
```elisp
(defun python/init-python ()
  (use-package python
    :defer t
    :mode (("SConstruct\\'" . python-mode) ("SConscript\\'" . python-mode))
    :init
    (progn
      (spacemacs/register-repl 'python
                               'spacemacs/python-start-or-switch-repl "python")
      (spacemacs//bind-python-repl-keys)
      (spacemacs/add-to-hook 'python-mode-hook
                             '(spacemacs//python-setup-backend
                               spacemacs//python-default))
```

Please also make sure that the default case, lsp not bound, is working as in the other layers, normally the behaviour should be when the variable is `nil` lsp is used if lsp layer is loaded otherwise the non lsp solution should be loaded.
This can be overwritten by setting the backend in the dotfile. There is normally a getter function for this `(spacemacs//layer-backend)` which must be used to obtain the backend in the layer.